cloud/google/machineactuator.go

```diff
@@ -719,6 +705,28 @@ func (gce *GCEClient) getImagePath(img string) (imagePath string) {
 	return defaultImg
 }
 
+func newDisks(config *gceconfigv1.GCEProviderConfig, zone string, imagePath string, minDiskSizeGb int64) []*compute.AttachedDisk {
```

Comment:
This min size should eventually be a validation error

Reply:
Yeah, min size was brought in recent time when we bumped up the min to 30 for custom images, and now for any image (as of a PR a few weeks ago). I was just trying to keep it as it was. I wonder if it should just be removed all together?